app/views/registered_applications/_form.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= form_for application, url: { action: "create" } do |f| %>
```

Comment:
@jeffminnear 

I was barking up the wrong tree with this one!

In `routes.rb`, use the `as: ...` option with the `resources` call:

```diff
- resources :registered_applications
+ resources :registered_applications, as: :applications
```

Then the form partial can use:

```diff
- <%= form_for application, url: { action: "create" } do |f| %>
+ <%= form_for application do |f| %>
```

See this section in the Rails official guide to routing: http://edgeguides.rubyonrails.org/routing.html#overriding-the-named-helpers